File: JustWeather/JustWeather/JustWeather/ViewController.m
//
//  ViewController.m
//  JustWeather
//
//  Created by Blake Oistad on 12/29/15.
//  Copyright © 2015 Blake Oistad. All rights reserved.
//

#import "ViewController.h"
#import "NetworkManager.h"
#import "DataManager.h"

@interface ViewController ()

@end

@implementation ViewController

#pragma mark - Interactivity Methods

- (IBAction)searchButtonPressed:(id)sender {
    NSLog(@"Search Button Pressed");
}

#pragma mark - Life Cycle Methods

- (void)viewDidLoad {
    [super viewDidLoad];
    NetworkManager *networkManager = [NetworkManager sharedInstance];
    DataManager *dataManager = [DataManager sharedInstance];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
}

@end

File: AudioC/audioTest3000/audioTester3000/audioTester3000/ViewController.swift
//
//  ViewController.swift
//  audioTester3000
//
//  Created by Blake Oistad on 12/24/15.
//  Copyright © 2015 Blake Oistad. All rights reserved.
//

import UIKit
import MediaPlayer
import AVFoundation

class ViewController: UIViewController {

    // MARK: - Properties
    
    let audioPlayer = AVAudioPlayer()
    
    // MARK: - Interactivity Methods
    
    @IBAction func playButtonPressed(sender: UIButton) {
        print("Play button pressed")
    }
    
    @IBAction func pauseButtonPressed(sender: UIButton) {
        print("Pause button pressed")
    }
    
    // MARK: - Life Style Methods
    
    override func viewDidLoad() {
        super.viewDidLoad()
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }


}

